Applications/Games/Space Colony/Steam/script.js

```diff
@@ -0,0 +1,22 @@
+include(["engines", "wine", "quick_script", "steam_script"]);
+include(["engines", "wine", "verbs", "dotnet40"]);
+include(["engines", "wine", "verbs", "vcrun2010"]);
+
+var installerImplementation = {
+    run: function () {
+        new SteamScript()
+            .name("Space Colony")
+            .editor("Firefly Studios")
+            .author("Zemogiter")
+            .appId(297920)
+            .preInstall(function(wine,/* wizard*/) {
+                .wine.dotnet40;
```

Comment:
`.wine.dotnet40;` is not valid Javascript.

Reply:
Have you tested this? What is `.wine` supposed to do?

Reply:
I tried to write the entire script from memory and ended up puting dots where they shouldn't be.

Reply:
So both scripts are tested and working now? To be honest, I'm a bit confused why the local script doesn't need dotnet40 and vcrun2010 while the Steam script needs it.

Reply:
Me too but the steam version won't work at all without those two dependencies. My guess is that Steam version displays a launcher with email subscription and language options and upon pressing "play" the actual game launches. There is no diffirence in file structure between steam and local instalations therefore dotnet40 and vcrun2010 are needed just for that launcher.

I was testing this script in July and it was working but yesterday it was throwing errors so I need to test it some more.